From a465cfdafc81ddf67fcf37b8517078cd3d63892a Mon Sep 17 00:00:00 2001
From: Gwendal Grignou <gwendal@chromium.org>
Date: Fri, 17 Jun 2022 09:09:18 -0700
Subject: [PATCH] fix assert logic

The logic value in the assert should be |false| (0) to hit.

---
 nih/tests/test_tree.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/nih/tests/test_tree.c b/nih/tests/test_tree.c
index 58f1178..2b8ccc2 100644
--- a/nih/tests/test_tree.c
+++ b/nih/tests/test_tree.c
@@ -1357,7 +1357,7 @@ my_filter (NihTree **nodes,
 	if (node == nodes['l' - 97])
 		return TRUE;
 
-	assert ("not reached");
+	assert (! "not reached");
 	return TRUE;
 }
 
-- 
2.31.0

